src/tests/c_guests/c_simpleguest/main.c

```diff
@@ -295,6 +303,9 @@ void hyperlight_main(void)
     HYPERLIGHT_REGISTER_FUNCTION("GuestAbortWithMessage", guest_abort_with_msg);
     HYPERLIGHT_REGISTER_FUNCTION("ExecuteOnStack", execute_on_stack);
     HYPERLIGHT_REGISTER_FUNCTION("LogMessage", log_message);
+    // HYPERLIGHT_REGISTER_FUNCTION macro does not work for functions that return VecBytes,
```

Comment:
Did you mean to remove this comment?

Reply:
no I left it on purpose to indicate why you cannot use HYPERLIGHT_REGISTER_FUNCTION